userprefs.php

```diff
@@ -166,19 +166,28 @@ function do_font_sample_update(font_index, size_index, layout) {
     // font_sample when they change.
     window.addEventListener('DOMContentLoaded', () => {
         for (const orientation of ['v', 'h']) {
-            document.querySelectorAll('input[name=\"' + orientation + '_fntf\"]').forEach(radio => {
-                radio.addEventListener('change', function () {
-                    do_font_sample_update(this.value, null, orientation);
+            let radios = document.querySelectorAll('input[name=\"' + orientation + '_fntf\"]');
+            if(radios) {
```

Comment:
*minor* other places we put a space after `if`, or `for` if we want to be consistent.

Reply:
I've changed this. It would be nice to make this a javascript file so we could lint it.

Reply:
Yeah. I wish lunters worked better on online scripts :/